docs/data-sources/vpc_private_network.md

```diff
@@ -21,17 +23,30 @@ data "scaleway_vpc_private_network" "my_id" {
 }
 ```
 
+### Regional
+
+```hcl
+# Get info by name
+data "scaleway_vpc_private_network" "my_name" {
+  name = "foobar"
+  is_regional = true
```

Comment:
Is there a way for us to guess it? I'm concerned because going forward we won't have zonal vpc anymore. therefore this field got good chance to be deprecated. In case there is an ambiguous PN that got both a version in zonal and regional maybe we should have a warning and call for the user to rename it?